modules/xmpp/JingleSessionPC.js

```diff
@@ -2524,32 +2581,30 @@ export default class JingleSessionPC extends JingleSession {
      * @see {@link _localVideoActive}
      */
     modifyContents(jingleContents) {
-        const newVideoSenders
-            = JingleSessionPC.parseVideoSenders(jingleContents);
-        const newMaxFrameHeight
-            = JingleSessionPC.parseMaxFrameHeight(jingleContents);
+        const newVideoSenders = JingleSessionPC.parseVideoSenders(jingleContents);
+        const newMaxFrameHeight = JingleSessionPC.parseMaxFrameHeight(jingleContents);
+        const sourceMaxFrameHeights = JingleSessionPC.parseSourceMaxFrameHeight(jingleContents);
 
         // frame height is optional in our content-modify protocol
         if (newMaxFrameHeight) {
             logger.info(`${this} received remote max frame height: ${newMaxFrameHeight}`);
             this.remoteRecvMaxFrameHeight = newMaxFrameHeight;
-            this.eventEmitter.emit(
-                MediaSessionEvents.REMOTE_VIDEO_CONSTRAINTS_CHANGED, this);
+            this.eventEmitter.emit(MediaSessionEvents.REMOTE_VIDEO_CONSTRAINTS_CHANGED, this);
```

Comment:
can we have two separate events? `MediaSessionEvents.REMOTE_VIDEO_CONSTRAINTS_CHANGED` for the old flow and `MediaSessionEvents.REMOTE_SOURCE_CONSTRAINTS_CHANGED` for the new flow?
[the code which is processing the event] is hard to follow (hard to tell why would there be no constraints)

[the code which is processing the event]:https://github.com/jitsi/lib-jitsi-meet/pull/1983/files#diff-88c6b4c65414c3f9776f316273316a331f84e1755c848039295038175ac8a9b1R78